templates/quickstart-hashicorp-vault.template

```diff
@@ -262,23 +533,19 @@ Resources:
           files:
             "/etc/cron.hourly/cloudwatch-monitoring.sh":
               source:
-                Fn::Sub:
-                - https://s3.amazonaws.com/${QSS3BucketName}/${QSS3KeyPrefix}scripts/cloudwatch-monitoring.sh
-                - QSS3BucketName:
-                    Ref: QSS3BucketName
-                  QSS3KeyPrefix:
-                    Ref: QSS3KeyPrefix
+                !Sub
+                  - 'https://${S3Bucket}.s3.${S3Region}.${AWS::URLSuffix}/${QSS3KeyPrefix}scripts/cloudwatch-monitoring.sh'
+                  - S3Region: !If [UsingDefaultBucket, !Ref 'AWS::Region', !Ref QSS3BucketRegion]
+                    S3Bucket: !If [UsingDefaultBucket, !Sub '${QSS3BucketName}-${AWS::Region}', !Ref QSS3BucketName]
```

Comment:
Can you replace this S3 path with an `inline` reference to the cron config itself? `*/5 * * * * sudo /usr/local/aws-scripts-mon/mon-put-instance-data.pl --mem-util --mem-used --from-cron`

Delete file `cloudwatch-monitoring.sh` file from the `scripts` folder as well.